EDDiscovery/3DMap/FormMap.cs

```diff
@@ -277,6 +279,8 @@ private ToolStripButton _AddExpedition(string name, Func<DateTime> starttime, Fu
 
         private void FillExpeditions()
         {
+            if (VisitedSystems == null || VisitedSystems.Count < 1)
```

Comment:
This means that if the visited system history is empty when the map is first opened, the history filter list will be empty until the map is reloaded.